src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -0,0 +1,348 @@
+          <div class="default-view">
+            ${this.renderTemplate('default', { person: this.personDetails }) ||
+              html`
+                <mgt-person .personDetails=${this.personDetails} .personImage=${this.personImage}></mgt-person>
+                <div class="details">
+                  <div class="display-name">${user.displayName}</div>
+                  ${jobTitle} ${department}
+                  <div class="base-icons">
+                    ${this.renderIcons()}
+                  </div>
+                </div>
+              `}
+          </div>
+          <div class="additional-details-container" @click=${this._showAdditionalDetails}>
+            ${this.renderAdditionalDetails()}
+          </div>
+        </div>
+      `;
+    }
+  }
+
+  private renderIcons() {
+    const user = this.personDetails;
+    // tslint:disable-next-line: one-variable-per-declaration
+    let chat, email, phone;
+    if ((user as MicrosoftGraph.User).mailNickname) {
+      chat = html`
+        <div @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState} @click=${this._chatUser}>
+          ${svgHelper.getSVG('chat', '#666666')}
+        </div>
+      `;
+    }
+    if (getEmailFromGraphEntity(user)) {
+      email = html`
+        <div @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState} @click=${this._emailUser}>
+          ${svgHelper.getSVG('email', '#666666')}
+        </div>
+      `;
+    }
+
+    if ((user as MicrosoftGraph.User).businessPhones && (user as MicrosoftGraph.User).businessPhones.length > 0) {
+      phone = html`
+        <div @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState} @click=${this._callUser}>
+          ${svgHelper.getSVG('phone', '#666666')}
+        </div>
+      `;
+    }
+
+    if (this.isExtended === true) {
+      return html``;
+    } else {
+      return html`
+        ${chat} ${email} ${phone}
+      `;
+    }
+  }
+
+  private renderAdditionalDetails() {
+    const user = this.personDetails;
+
+    // tslint:disable-next-line: one-variable-per-declaration
+    let phone, email, location, chat;
+
+    if ((user as MicrosoftGraph.User).businessPhones && (user as MicrosoftGraph.User).businessPhones.length > 0) {
+      phone = html`
+        <div class="details-icon" @click=${this._callUser}>
+          ${svgHelper.getSVG('phone-small', '#666666')}
+          <span class="link-subtitle data" @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState}
+            >${(user as MicrosoftGraph.User).businessPhones[0]}</span
+          >
+        </div>
+      `;
+    }
+
+    if (getEmailFromGraphEntity(user)) {
+      email = html`
+        <div class="details-icon" @click=${this._emailUser}>
+          ${svgHelper.getSVG('email-small', '#666666')}
+          <span class="link-subtitle data" @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState}
```

Comment:
these mouseover and mouseout events should be handled in CSS instead https://developer.mozilla.org/en-US/docs/Web/CSS/:hover

Reply:
Added a new class for the base icons, and removed the old functionality:

554db18